apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/cohorts/index.vue

```diff
@@ -0,0 +1,255 @@
+<script setup lang="ts">
+const route = useRoute();
+const router = useRouter();
+const config = useRuntimeConfig();
+const pageSize = 10;
+
+const currentPage = ref(1);
+if (route.query?.page) {
+  const queryPageNumber = Number(route.query?.page);
+  currentPage.value =
+    typeof queryPageNumber === "number" ? Math.round(queryPageNumber) : 1;
+}
+let offset = computed(() => (currentPage.value - 1) * pageSize);
+
+let filters = reactive([
+  {
+    title: "Search in cohorts",
+    columnType: "_SEARCH",
+    search: "",
+    searchTables: ["collectionEvents", "subcohorts"],
+    initialCollapsed: false,
+  },
+  {
+    title: "Areas of information",
+    refTable: "AreasOfInformation",
+    columnName: "areasOfInformation",
+    columnType: "ONTOLOGY",
+    filterTable: "collectionEvents",
+    conditions: [],
+  },
+  {
+    title: "Data categories",
+    refTable: "DataCategories",
+    columnName: "dataCategories",
+    columnType: "ONTOLOGY",
+    filterTable: "collectionEvents",
+    conditions: [],
+  },
+  {
+    title: "Population age groups",
+    refTable: "AgeGroups",
+    columnName: "ageGroups",
+    columnType: "ONTOLOGY",
+    filterTable: "collectionEvents",
+    conditions: [],
+  },
+  {
+    title: "Sample categories",
+    refTable: "SampleCategories",
+    columnName: "sampleCategories",
+    columnType: "ONTOLOGY",
+    filterTable: "collectionEvents",
+    conditions: [],
+  },
+]);
+
+let search = computed(() => {
+  // @ts-ignore
+  return filters.find((f) => f.columnType === "_SEARCH").search;
+});
+
+const query = computed(() => {
+  return `
+  query Cohorts($filter:CohortsFilter, $orderby:Cohortsorderby){
```

Comment:
don't you want gql autoformat?

Reply:
yea, it started out as really small request , single line , i'll add the gql

Reply:
added

Reply:
and reverted, has issue with nuxt fetch , need to look into with this does not work in this context